repository-management.md

```diff
@@ -0,0 +1,101 @@
+# Repository Management
+
+## Package Names
+
+Package names for the official `wp-cli` GitHub repository need to be discussed upfront with the [maintainers](https://github.com/orgs/wp-cli/teams/maintainers).
+
+Package names for WP-CLI commands must end with the `-command` suffix.
+
+_Example: `wp-cli/scaffold-package-command`._
+
+## Package Versions
+
+Package version follow the [Semantic Versioning Specification (SemVer)](http://semver.org/).
```

Comment:
In layman's terms for us, this typically means:

* Patch releases (e.g. `1.0.x`) are used for bug fixes, documentation changes, small enhancements, etc.
* Minor releases  (e.g. `1.x.0`) are used when a new command is introduced to a package, or a major enhancement is made to an existing command.

Reply:
https://github.com/wp-cli/handbook/pull/127/commits/20a23d4c18f260317ed24ed70245463b60537ff8 & https://github.com/wp-cli/handbook/pull/127/commits/e9efb1a4ce74687863cb29debb0b12f6a23d6696